docs/migration-pulp-installer-to-multi-process.md

```diff
@@ -0,0 +1,286 @@
+
+This is necessary for container networks to access PostgreSQL.
+
+Ensure this line is in `/var/lib/pgsql/data/pg_hba.conf` (Fedora 36) / `/etc/postgresql/14/main/pg_hba.conf` (Ubuntu 22.04):
+```
+host all all 0.0.0.0/0 md5
+```
+
+Ensure this line is in `/var/lib/pgsql/data/postgresql.conf` (Fedora 36) / `/etc/postgresql/14/main/postgresql.conf` (Ubuntu 22.04):
+```
+listen_addresses = '*'
+```
+
+Run:
+```
+sudo systemctl restart postgresql
+```
+
+### Dump the PostgreSQL database (EL7 and EL8 only)
+
+```
+sudo -u postgres pg_dumpall > /tmp/dump.sql
+```
+
+Verify that this command outputs "0", which indicates that the dump was successful.
+
+```
+echo $?
+```
+
+```
+sudo mv /tmp/dump.sql /var/lib/pgsql
+```
+
+### Disable/restart the third-party services
+
+On Fedora 36:
+```
+sudo systemctl disable --now nginx redis
+```
+On Ubuntu 22.04:
+```
+sudo systemctl disable --now nginx redis-server
+```
+On on EL, or Fedora prior to 36:
+```
+sudo systemctl disable --now postgresql nginx redis
+```
+On Debian 11:
+```
+sudo systemctl disable --now postgresql nginx redis-server
+```
+
+
+### Manage the Pulp storage directory
+
+1st, move the directory:
+
+```
+sudo mv /var/lib/pulp/ pulp_storage
+```
+
+Next change ownership of the directory:
+
+If running rootless podman:
+```
+sudo chown -R $USER:$(id -gn) pulp_storage
+podman unshare chown -R 700:700 pulp_storage
+```
+
+If running podman as root, or docker:
+```
+sudo chown -R 700:700 pulp_storage
+```
+
+### Manage the Pulp configuration directory
+
+1st, move the directory:
+```
+sudo mv /etc/pulp settings
+```
+
+Next change ownership of the directory:
+
+If running rootless podman:
+```
+sudo chown -R $USER:$(id -gn) settings
+podman unshare chown -R 700:700 settings
+```
+If running podman as root, or docker:
+```
+sudo chown -R 700:700 settings
+```
+
+### Configure Pulp to talk to the database (Fedora 36 and Ubuntu 22.04 only)
+
+If running podman, modify `settings/settings.py` so that "DATABASE" includes:
+```
+'HOST': 'host.containers.internal'
+```
+An example of what the line will look like:
+```
+DATABASES = {'default': {'HOST': 'host.containers.internal', 'ENGINE': 'django.db.backends.postgresql', 'NAME': 'pulp', 'USER': 'pulp', 'PASSWORD': 'pulp'}}
+```
+
+If running docker, modify `settings/settings.py` so that "DATABASE" includes:
+```
+'HOST': 'host.docker.internal'
+```
+An example of what the line will look like:
+```
+DATABASES = {'default': {'HOST': 'host.docker.internal', 'ENGINE': 'django.db.backends.postgresql', 'NAME': 'pulp', 'USER': 'pulp', 'PASSWORD': 'pulp'}}
+```
+
+### Manage the PostgreSQL data directory (EL, Fedora prior to 36, and Debian 11 only)
+
+1st, move the directory:
+
+If on EL, or Fedora prior to 36
+```
+sudo mv /var/lib/pgsql pgsql
+```
+If on Debian 11:
+```
+sudo mv /var/lib/postgresql/13 pgsql
+sudo mv pgsql/main pgsql/data
+```
+If on EL7 or EL8:
+```
+sudo mv /var/lib/pgsql/data /var/lib/pgsql/data_old
```

Comment:
This moves the data to `/var/lib/pgsql/data_old`  but the following commands assume it's in `./pgsql`

Reply:
This should be a relative path and not absolute path.